src/modules/recipes/FeaturedRecipes/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect } from "react";
+import {
+  useRecipes,
+  useRecipesDispatch,
+  RECIPE_ACTIONS,
+} from "../RecipesProvider";
+import { fetchRecipesByIngredient } from "../recipeService";
+
+export const FeaturedRecipes = () => {
+  const recipes = useRecipes();
+  const dispatch = useRecipesDispatch();
+
+  useEffect(() => {
+    fetchRecipesByIngredient("chicken_breast").then((recipes) =>
+      dispatch({ type: RECIPE_ACTIONS.update, payload: recipes })
+    );
+  }, [dispatch]);
+
+  // use RecipeList component to display the list of recipe below
+
+  const handleRecipeClick = (idMeal) => {
+    console.log("Recipe clicked:", idMeal);
+  };
+
+  return (
+    <>
+      {recipes.length > 0 ? (
```

Comment:
The code here repeats RecipeList component. Instead, make sure that the RecipeList only gets recipes from Context and display whatever is there on the page.

On the other hand, FeaturedRecipes component should fetch recipes and then just display RecipeList component.